src/main/java/com/github/manolo8/darkbot/backpage/auction/AuctionItems.java

```diff
@@ -0,0 +1,86 @@
+package com.github.manolo8.darkbot.backpage.auction;
+
+public class AuctionItems {
+    protected int auctionType; //0=hour, 1=day, 2=hour
```

Comment:
could probably be turned into an enum for simpler use, but if you don't feel like changing it, it can stay